server/lib/nl_data_spec.py

```diff
@@ -116,6 +117,14 @@ def compute(query_detection: Detection, context):
   main_place_name = places_detected.main_place.name
   main_place_type = places_detected.main_place.place_type
 
+  context_sv = None
+  context_place = None
+  if context and context['debug'] and context['debug']['data_spec']:
+    context_sv = context['debug']['data_spec']['primary_sv']
```

Comment:
Shouldn't we check for existence of 'primary_sv' key?

Reply:
Done